tests/fixtures/name-matches-to-wikidata.json

```diff
@@ -0,0 +1,96 @@
+{
+  "compareFunction": "name-matches-to-wikidata",
+  "fixtures": [
+    {
+      "description": "Test OSM name different from Wikidata name",
+      "expectedResult": {
+        "result:name_matches_to_wikidata": false
+      },
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "osm:id": 427818536,
+          "osm:type": "way",
+          "name": "Central Park is now something else",
+          "wikidata": "Q160409",
+          "osm:version": 1
+        },
+        "geometry": null
+      },
+      "oldVersion": null
+    },
+    {
+      "description": "Test OSM name matches with aliases on Wikidata",
+      "expectedResult": {},
```

Comment:
@bkowshik @planemad That makes sense. But need to find a good way to deal with parsing these different kind of results and scale it as well.